upgrade-2.x.sh

```diff
@@ -124,7 +133,7 @@ function upgradeToReleaseSupervisor() {
                     log "Running supervisor updater..."
                     progress 90 "ResinOS: running supervisor update..."
                     update-resin-supervisor
-                    stop_services
+                    remove_containers
```

Comment:
NIT: I would find it a little clearer if I had here both stop and remove functions invoked. I understand the dependency which made you code it this way though.

Reply:
Updated!